gallery/src/styles/NxTable/NxTableSortableExample.tsx

```diff
@@ -12,12 +12,14 @@ const NxTableSortable = () =>
   <table className="nx-table">
     <thead>
       <tr className="nx-table-row nx-table-row--header">
-        <th className="nx-cell nx-cell--header nx-cell--sortable">
-          <span>Name</span>
-          <span className="nx-cell__sort-icons fa-layers">
-            <NxFontAwesomeIcon icon={faSortUp} />
-            <NxFontAwesomeIcon icon={faSortDown} />
-          </span>
+        <th className="nx-cell nx-cell--header nx-cell--sortable" aria-sort="none">
+          <button type="button" className="nx-cell__sort-btn" aria-label="Name unsorted">
```

Comment:
Safari issues: 
<img width="189" alt="Screen Shot 2021-04-28 at 11 15 30 AM" src="https://user-images.githubusercontent.com/1991443/116428656-1da30500-a813-11eb-86aa-0da7a72fd8ae.png">

Solution is likely the same as the chevron button above.

Reply:
It's worse, the sort icons are gone.

Reply:
> It's worse, the sort icons are gone.

Checked on two computers, both running Safari 14.1 and I can lo longer reproduce.